sdk/cosmos/live-thinclient-platform-matrix.json

```diff
@@ -0,0 +1,25 @@
+{
+  "displayNames": {
+    "-Pthinclient": "ThinClient",
+    "Session": "",
+    "ubuntu": "",
+    "@{ enableMultipleWriteLocations = $true; defaultConsistencyLevel = 'Session'; enableMultipleRegions = $true }": "ThinClient"
```

Comment:
This is unnecessary the arm template params won't show up in the job title/display name since they are nested within MultiMaster_MultiRegion (which will be the title instead)